utility/test/actor_test.go

```diff
@@ -215,21 +223,14 @@ func TestUtilityContext_BeginUnstakingMaxPaused(t *testing.T) {
 			require.NoError(t, err, "error beginning unstaking max paused actors")
 
 			status, err := ctx.GetActorStatus(actorType, addrBz)
+			require.NoError(t, err)
 			require.Equal(t, typesUtil.UnstakingStatus, status, "actor should be unstaking")
+
 			test_artifacts.CleanupTest(ctx)
 		})
 	}
 }
 
-func TestUtilityContext_CalculateRelays(t *testing.T) {
```

Comment:
Why was this test removed?

Reply:
It was actually moved to a different part of the file, but I reverted it for clarity.